framework/Models/Code.py

```diff
@@ -101,25 +101,26 @@ def specializeValidateDict(cls):
     cls.validateDict['Input'  ][0]['required'    ] = False
     cls.validateDict['Input'  ][0]['multiplicity'] = 'n'
 
-  def __init__(self,runInfoDict):
+  def __init__(self, runInfoDict):
     """
       Constructor
       @ In, runInfoDict, dict, the dictionary containing the runInfo (read in the XML input file)
       @ Out, None
     """
     Model.__init__(self,runInfoDict)
-    self.executable         = ''   #name of the executable (abs path)
-    self.preExec            = None   #name of the pre-executable, if any
-    self.oriInputFiles      = []   #list of the original input files (abs path)
-    self.workingDir         = ''   #location where the code is currently running
-    self.outFileRoot        = ''   #root to be used to generate the sequence of output files
-    self.currentInputFiles  = []   #list of the modified (possibly) input files (abs path)
-    self.codeFlags          = None #flags that need to be passed into code interfaces(if present)
-    self.printTag           = 'CODE MODEL'
-    self.createWorkingDir   = True
-    self.foundExecutable    = True # True indicates the executable is found, otherwise not found
-    self.foundPreExec       = True # True indicates the pre-executable is found, otherwise not found
-    self.maxWallTime        = None # If set, this indicates the maximum CPU time a job can take.
+    self.executable = ''         # name of the executable (abs path)
```

Comment:
I truly prefer the way it was, even if it is not the pythonic way. It is a lot more readable.:)

Reply:
Yeah, I have mixed feelings, so I went with the standards in the end.